components/infobox/wikis/pubg/infobox_league_custom.lua

```diff
@@ -167,4 +172,15 @@ function CustomLeague:_getPlatform()
 	return _PLATFORMS[platform] or _PLATFORMS['default']
 end
 
+function CustomLeague._getPatchVersion()
+	if String.isEmpty(_args.patch) then return nil end
+	local content = PageLink.makeInternalLink(_args.patch, 'Patch ' .. _args.patch)
+	if not String.isEmpty(_args.epatch) then
+		content = content .. '&nbsp;&ndash;&nbsp;'
```

Comment:
make it const vars
why split it into 2 lines with both `content = content .. `?
would also be cleaner to do a return here

```lua
	if String.isNotEmpty(_args.epatch) then
		return content .. NONE_BREAKING_SPACE .. DASH .. NONE_BREAKING_SPACE
			.. PageLink.makeInternalLink(_args.epatch, 'Patch ' .. _args.epatch)
	end

	return content
```

Reply:
How should I type the nbsp and dash into the const? 

![image](https://github.com/Liquipedia/Lua-Modules/assets/88981446/ade56119-d4e3-48a4-8d70-0f18f04e10ba)
(I've try add symbols that replicates the replaced **&nbsp;&ndash;&nbsp;** line it will return unexpected symbol)

Reply:
- why the brackets?
- `local DASH = dash` makes it nil, use `local DASH = '&ndash;'`